src/App.vue

```diff
@@ -132,17 +169,24 @@ import { useToast } from "vue-toastification";
 
 export default {
   components: {
-    WorkspaceSwitcher,
     LocaleSwitcher,
-    IconButton,
     SharePlioDialog,
     EmbedPlioDialog,
+    WorkspaceSwitcher,
+    IconButton,
   },
   data() {
     return {
       showLanguagePickerDialog: false, // whether to show a language picker dialog box
       toast: useToast(), // use the toast component
       userClickedLogout: false, // if the user has clicked the logout button
+      // class for the create button
+      createButtonClass:
+        "bg-primary hover:bg-primary-hover rounded-lg w-full ring-primary p-2 sm:py-4",
+      isMenuButtonPressed: false,
```

Comment:
```suggestion
      isMenuButtonActive: false,
```
Pressed feels like ki it's continuously pressed, like I've continuously kept my mouse clicked at it. Active seems more intuitive to me